website/src/routes/api/(schemas)/string/index.mdx

```diff
@@ -1,7 +1,109 @@
 ---
 title: string
+description: Creates a string schema.
 ---
 
+import { Link } from '@builder.io/qwik-city';
+import { Property } from '~/components';
+
 # string
 
-> The content of this page is not yet ready. Until then just use the [source code](https://github.com/fabian-hiller/valibot/blob/main/library/src/schemas/string/string.ts).
+Creates a string schema.
+
+```ts
+// String schema with an optional pipe
+const Schema = string(pipe);
+
+// String schema with an optional message and pipe
+const Schema = string(message, pipe);
+```
+
+## Arguments
+
+- `message` <Property {...properties.message} />
+- `pipe` <Property {...properties.pipe}/>
+
+## Return
+
+- `Schema` <Property {...properties.stringSchema} />
+
+## Examples
+
+With `string` you validate the data type of the incoming data and with the `pipe` you can then validate all further details. In the event of an error, the `message` argument allows you to customize the error message.
+
+```ts
```

Comment:
Would it be interesting to add 1 example whit the error message?

Reply:
Yes, good idea! Where or how would you add it?

Reply:
Maybe changing line 52 by
`const UrlSchema = string("A URL must be of type string",[url('The URL is badly formatted.')]);`